projects/gen-ai.md

```diff
@@ -123,9 +123,9 @@ Other languages are not in the scope (yet).
 
 ## Meeting Times
 
-We'll use GenAI LLM Semantic Convention meetings until more suitable time for new participants is determined:
+We'll use GenAI Semantic Convention meetings until more suitable time for new participants is determined:
 
-- **Every week**: Wednesdays at 10 AM PST - Looking for alternative time to accommodate more time zones - https://strawpoll.com/e6Z2ApEKXgN
+- **Every other week**: Thursday at 10 AM PST
 - **Every other week**: Thursday 14:00 UTC+8
```

Comment:
If there is no push back to the proposal, can we update this to Thursday at 16:00 PST.

Reply:
yep, let's bring it up again tomorrow and give folks a bit of time to respond :)

Reply:
let's do it as a follow up